users/management/commands/set_ad_group_mappings.py

```diff
@@ -0,0 +1,63 @@
+from django.core.management import BaseCommand
+from helusers.models import ADGroup, ADGroupMapping
+
+# Group order:
+BERTH_SERVICES = 0
+BERTH_HANDLER = 1
+BERTH_SUPERVISOR = 2
+HARBOR_SERVICES = 3
+
+AD_GROUP_MAP = {
+    BERTH_SERVICES: {
+        "display_name": "Berth services",
+        "ad_group_name": "venepaikka_paakayttaja",
+    },
+    BERTH_SUPERVISOR: {
+        "display_name": "Berth supervisor",
+        "ad_group_name": "venepaikka_katselija",
+    },
+    BERTH_HANDLER: {
+        "display_name": "Berth handler",
+        "ad_group_name": "venepaikka_kasittelija",
+    },
+    HARBOR_SERVICES: {
+        "display_name": "Harbor services",
+        "ad_group_name": "venepaikka_satamapalvelu",
+    },
+}
+
+
+class Command(BaseCommand):
+    help = "Sets predefined AD group mappings for the predefined Venepaikka groups"
+
+    def handle(self, *args, **options):
+        # Clean all Venepaikka-related ADGroups, this also removes the ADGroupMappings
+        ADGroup.objects.filter(
+            name__icontains="sg_kuva_ad-tunnistamo_venepaikka_"
+        ).delete()
+
+        new_groups = 0
+        new_mappings = []
+
+        for group_id, ad_group_details in AD_GROUP_MAP.items():
+            display_name, ad_group_name = ad_group_details.values()
+
+            full_name = f"sg_kuva_ad-tunnistamo_venepaikka_{ad_group_name}"
```

Comment:
still double `venepaikka` in the end 👀

Reply:
For some reason my changes weren't pushed. 🤔

Reply:
Removed the second `venepaikka` from the [`ad_group_name`](https://github.com/City-of-Helsinki/berth-reservations/pull/197/files#diff-e4c0b5e01de0efe34408287d9cd55bf2R11)